### ISSUE TITLE
AutomationProperty doesn't work for TextCell

### ISSUE THREAD

**Issue by alexandergoncharov** [xamarin/Xamarin.Forms#3296]
### Description

AutomationProperty doesn't work for TextCell element. 

### Steps to Reproduce

1. Create TextCell. 
2. Add AutomationProperty to this TextCell
3. Add this TextCell to TableSection.
4. Run app on Iphone.
5. Use VoiceOver for checking this AutomationProperty.

Attached testProject (Check MainPage.xaml.cs): 
[test.zip](https://github.com/xamarin/Xamarin.Forms/files/2193329/test.zip)

### Expected Behavior
All AutomationProperty pronouncing successfully. 

### Actual Behavior
AutomationProperty isn't pronounced. It is like there isn't any AutomationProperty.

### Basic Information

- Version with issue: Xamarin.iOS Version: 11.12.0.4 (Visual Studio Enterprise)
- Last known good version:
- IDE: Visual Studio Enterprise 2017 for Mac Version 7.5.3 (build 7)
- Platform Target Frameworks: 
  - iOS:  10.3.3, 11.4
- Nuget Packages:
- Affected Devices: Iphone 5, Iphone X
